src/hats/io/file_io/file_io.py

```diff
@@ -269,9 +269,7 @@ def delete_file(file_handle: str | Path | UPath):
     file_handle.unlink()
 
 
-def read_parquet_file_to_pandas(
-    file_pointer: str | Path | UPath, file_open_kwargs: dict = None, **kwargs
-) -> pd.DataFrame:
+def read_parquet_file_to_pandas(file_pointer: str | Path | UPath, **kwargs) -> pd.DataFrame:
```

Comment:
Does anyone know what `file_open_kwargs` is/was being used for? I looked through hats, hats-import, lsdb, and hats-cloudtests and I don't see it being used in any of the calls to this function. Since removing `file_pointer.open` means that this function doesn't do anything with it anymore, I've taken it out here. If something/someone does need it, I'd need to know what the use case is in order to figure out where/how to put it back in.